src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -37,6 +38,14 @@ export const ProgressStep = ({ ...props }) => {
     [className]: className,
   });
 
+  const handleKeyDown = e => {
+    const key = e.key || e.which;
+
+    if (key === 'Enter' || key === 13 || key === ' ' || key === 32) {
```

Comment:
we may be able to use our [keyboard event handler helpers](https://github.com/IBM/carbon-components-react/blob/master/src/tools/key.js) here